### PR TITLE
Adding template en français

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Thank you, @phrawzty & @jooooooon for https://2019.devopsrex.fr/content/code-de-conduite and https://github.com/devopsdays/devopsdays-web/pull/9164! I am attempting to add a reasonable template en français based on your devops rex code of conduct. I'll note a few questions inline.

### CODE REVIEW

File: utilities/examples/content/events/yyyy-city/conduct-fr.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
++++
+Title = "Conduct"
+Type = "event"
+Description = "Code de conduite devopsdays CITY YYYY"
```
Comment (bridgetkromhout) [inline utilities/examples/content/events/yyyy-city/conduct-fr.md]: Is this correct?

File: utilities/examples/content/events/yyyy-city/conduct-fr.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
++++
+
+Tous les participants, conférenciers, sponsors et volontaires à devopsdays CITY YYYY doivent accepter le code d'éthique et de déontologie, ou « code de conduite » suivant. Les organisateurs s’attacheront à faire respecter ce code durant l’événement. Nous attendons la coopération de chacun‧e pour assurer un environnement sain pour tous.
```
Comment (bridgetkromhout) [inline utilities/examples/content/events/yyyy-city/conduct-fr.md]: > à devopsdays CITY YYYY

Does this make sense? (I'm not sure where we say à or don't.)

File: utilities/examples/content/events/yyyy-city/conduct-fr.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Nous ne tolérons aucun harcèlement des participant‧e‧s à la conférence, sous quelque forme que ce soit, quelles qu’en soient les circonstances. Les expressions et les images à connotation sexuelle ne sont pas appropriées lors de l'événement. Ceci inclut les conférences, les ateliers, les soirées, Twitter et les autres médias en ligne.
+
+L'organisation s'engage à prendre des sanctions appropriées pour les personnes qui violent ces règles, jusqu'à l'exclusion sans remboursement et le recours légal.
+
+
+### La Version Moins Rapide
+
+Le harcèlement comprend les commentaires à l'oral ou à l'écrit ainsi que les  gestes offensants sur le sexe, l'identité sexuelle, l'âge, l'orientation sexuelle, le handicap, l'apparence physique, le poids, la race, l'ethnie, la religion, les choix technologiques, les images à connotation sexuelle dans des lieux publics, les intimidations délibérées, la traque, la poursuite, un harcèlement photographique ou vidéo, une suite d'interruptions des conférences et des autres sessions, un contact physique inapproprié et des avances sexuelles non désirées. sans son consentement ou à son insu.
```
Comment (bridgetkromhout) [inline utilities/examples/content/events/yyyy-city/conduct-fr.md]: I noticed that the French template is missing `les choix technologiques` in the less-short version, but probably we want it here too?

File: utilities/examples/content/events/yyyy-city/conduct-fr.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Face à un comportement de harcèlement, l'équipe d'organisation de la conférence peut prendre toute action qui lui semble adéquate. Cela va d'un simple avertissement à l'exclusion sans remboursement de la conférence.
+
+Si vous vous sentez harcelé‧e, si vous pensez que quelqu'un se fait harceler, et plus généralement en cas de problème, merci de contacter immédiatement l’équipe d’organisation de l'événement.
+
+Les membres de l'organisation sont facilement identifiables grâce à un badge "STAFF". Les membres de l'organisation seront ravi‧e‧s d'aider les participants à contacter la sécurité de l'événement, ou les forces de l'ordre ; à fournir une escorte ainsi qu'à aider de toute autre façon les personnes victimes de harcèlement, pour garantir leur sécurité pendant la durée de l'événement. Nous apprécions votre participation à l'événement.
```
Comment (bridgetkromhout) [inline utilities/examples/content/events/yyyy-city/conduct-fr.md]: I figure we shouldn't assume all events will print STAFF shirts, but they probably will all have it on the badge. Is `sont facilement identifiables grâce à un badge "STAFF"` reasonable?

**Comment by bridgetkromhout** [comment]
Closing this until we have time to revisit it.

**Comment by bridgetkromhout** [comment]
@phrawzty can you take a look at this for template purposes? Thanks!

File: utilities/examples/content/events/yyyy-city/conduct-fr.md (hunk 1 above)
Comment (phrawzty) [reply utilities/examples/content/events/yyyy-city/conduct-fr.md]: That is correct.

File: utilities/examples/content/events/yyyy-city/conduct-fr.md (hunk 3 above)
Comment (phrawzty) [reply utilities/examples/content/events/yyyy-city/conduct-fr.md]: Agreed—that's an error on our part. cc @jooooooon .

**Pull request merged**